arch/arm/src/cxd56xx/cxd56_dmac.c

```diff
@@ -400,6 +400,11 @@ static void _dmac_intc_handler(int ch)
   int itc;
   int err;
 
+  if (dev == NULL)
+    {
+      return;
```

Comment:
Please include a debug error message, because this fuction returns void, so there is not direct way to know it failed.